tools/milestone.py

```diff
@@ -0,0 +1,117 @@
+"""Script for automatically creating a milestone in repositories.
+
+Only specified repositories (via environment variable) will be handled.
+"""
+
+import datetime
+import os
+
+import github
+
+# =============================================================================
+#
+# The purpose of this code is to connect to an org, connect to a repository
+# and create a milestone. This milestone will be associated to a certain date
+# provided as an input argument.
+#
+# =============================================================================
```

Comment:
Thanks for documenting the behaviour of this code. As a suggestion, consider moving these lines to the docstring of the script. The reason is that those will get rendered for sure when generating any documentation.

Reply:
Agreed! =)